frontend/packages/core/src/Input/stories/radio.stories.tsx

```diff
@@ -25,3 +25,10 @@ export const Disabled = Template.bind({});
 Disabled.args = {
   disabled: true,
 };
+
+export const NameValueAndOnChange = Template.bind({});
+NameValueAndOnChange.args = {
+  name: "foo",
+  value: "bar",
+  onChange: e => e.target.value,
```

Comment:
`onChange` can be added as an [Action arg](https://storybook.js.org/docs/react/essentials/actions#action-argtype-annotation). example: https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/Input/stories/checkbox.stories.tsx#L10-L12

Reply:
thanks! just updated